Update server.md (#27484)
DataDog/documentation: content/en/real_user_monitoring/browser/setup/server.md

## Prerequisites

> For Auto-Instrumentation, ensure the [Datadog Agent][2] is installed with APM enabled and that your application is instrumented with the Datadog tracing library.

Reviewer: ```suggestion
For Auto-Instrumentation, ensure the [Datadog Agent][2] is installed with APM enabled.
```
There is no dependency with tracing libraries.

Author: Yeah got clarification that it was a Datadog agent issue, not a dependency issue. will delete this request!

> The automatic installation method requires that you have the [Datadog Agent][2] installed with APM enabled. Your application has been instrumented with the Datadog tracing library.

Reviewer 2: ```suggestion
For Auto-Instrumentation, ensure the [Datadog Agent][2] is installed with APM enabled and that your application is instrumented with the Datadog tracing library.
```
